warehouse/src/main/java/com/visma/warehouse/security/SecurityConfigDatabase.java

```diff
@@ -25,11 +35,21 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 
     @Bean
     public InMemoryUserDetailsManager userDetailsService() {
-        UserDetails user = User
-                .withUsername("user")
-                .password("{noop}password")
+
+        List<UserDetails> shops = shopRepository
```

Comment:
Labai neefektyvu uzkrauti visus userius, be to jei db atsirastu naujas, neatsinaujintu be restart. Pabandyk tiesiog implemetuoti UserDetailsService